functions/__fzf_display_value_or_error.fish

```diff
@@ -1,10 +0,0 @@
-# helper function for __fzf_search_shell_variables
```

Comment:
Lol I think you basically touched 90% of my original code 😛 
Not a problem but I find it kinda funny.